src/lib/select/select-header.html

```diff
@@ -0,0 +1,3 @@
+<span cdkTrapFocus>
```

Comment:
Do we always want to focus trap here? What's your reasoning for adding it by default instead of letting the user add it?

Reply:
I'd imagine that we do, just because the primary use case is having a search field. Also it removes a little bit of boilerplate for the consumer.